components/frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "quality-time-app",
   "version": "5.0.1",
   "private": true,
-  "proxy": "http://127.0.0.1:5001",
+  "proxy": "http://0.0.0.0:5001",
```

Comment:
This can only work if it's a listen address, but I assume it isn't?

Reply:
I was getting error messages from node, saying it couldn't proxy requests, but I can't reproduce it now, so reverting this change.